lib/view_component/base.rb

```diff
@@ -52,7 +52,9 @@ def render_in(view_context, &block)
       self.class.compile(raise_errors: true)
 
       @view_context = view_context
-      @lookup_context ||= view_context.lookup_context
+      @lookup_context ||= view_context.lookup_context.tap do |c|
+        c.view_paths.unshift(Rails.root.join("app/components"))
```

Comment:
Is it possible to do this at the Railtie level instead of the component level?

I'm a little concerned about performance since this would run at least once per-component render.

Reply:
I cached `lookup_context` in class attribute, bun it work only rails 6.0, 6.1, older versions is a hard work for support